### PR TITLE
support timeout of local-app

### PR DESCRIPTION
**Pull request by mustard-mh**
This PR fixes https://github.com/gitpod-io/gitpod/issues/7074

### CODE REVIEW

File: extensions/gitpod/src/extension.ts [hunk 1]
```diff
@@ -231,7 +231,8 @@ export async function activate(context: vscode.ExtensionContext) {
 					GITPOD_LCA_API_PORT: String(apiPort),
 					GITPOD_LCA_AUTO_TUNNEL: String(false),
 					GITPOD_LCA_AUTH_REDIRECT_URL: `${vscode.env.uriScheme}://${context.extension.id}${authCompletePath}`,
-					GITPOD_LCA_VERBOSE: String(vscode.workspace.getConfiguration('gitpod').get<boolean>('verbose', false))
+					GITPOD_LCA_VERBOSE: String(vscode.workspace.getConfiguration('gitpod').get<boolean>('verbose', false)),
+					GITPOD_LCA_SSH_IDLE_TIMEOUT: String(vscode.workspace.getConfiguration('gitpod').get<string>('ssh-idle-timeout', '3h'))
```
Comment (akosyakov) [inline extensions/gitpod/src/extension.ts:235]: it should be `sshIdleTimeout`, using dash separated names is alien in VS Code settings

Comment (mustard-mh) [reply extensions/gitpod/src/extension.ts]: updated

Comment (akosyakov) [reply extensions/gitpod/src/extension.ts]: i think we can go back to `timeout` and merge https://github.com/gitpod-io/gitpod/pull/7372 instead

sorry for back and forth

Comment (mustard-mh) [reply extensions/gitpod/src/extension.ts]: updated, use `timeout` instead

**Review by akosyakov — APPROVED** [review]
lgtm

**Pull request merged**